AutoPartsShop.API/Controllers/ProductsController.cs

```diff
@@ -0,0 +1,140 @@
+﻿using AutoPartsShop.API.Extensions;
+using AutoPartsShop.API.Repositories.Interfaces;
+using AutoPartsShop.Models.DTOs;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AutoPartsShop.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ProductsController : ControllerBase
+    {
+        private readonly IProductRepository productRepository;
+
+
+        public ProductsController(IProductRepository productRepository)
+        {
+            this.productRepository = productRepository;
+        }
+
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<ProductDTO>>> GetItems()
```

Comment:
The methods name should reflect that we are working with products. So, `GetItems` should become `GetProducts`, `GetItem` should become `GetProduct` and so on.